docker/common/install_cuda_toolkit.sh

```diff
@@ -12,6 +12,18 @@ NVCC_VERSION_OUTPUT=$(nvcc --version)
 OLD_CUDA_VER=$(echo $NVCC_VERSION_OUTPUT | grep -oP "\d+\.\d+" | head -n 1)
 echo "The version of pre-installed CUDA is ${OLD_CUDA_VER}."
 
+check_cuda_version() {
+    if [ -z "$CUDA_VERSION" ] && [ -z "$CUDA_DRIVER_VERSION" ]; then
```

Comment:
```suggestion
    if [ -n "$CUDA_VERSION" ] && [ -n "$CUDA_DRIVER_VERSION" ]; then
```

I think you should use `-n`, e.g., checking if both variables are non-empty.